GETTING_STARTED.md

```diff
@@ -0,0 +1,61 @@
+# Getting started
+This is a guide on how to quickly start using the SDK to build your idea, so it is more a guide for clients of the SDK
+rather than for contributors to the SDK, although anyone can benefit from it.
+
+> **Warning**
+> - The steps below are only suitable for development environments.
+> - Postgres username and password are [placeholders](./packages/cardano-services/placeholder-secrets) and should be changed.
+
+## Project Overview
+The project is divided into two main parts:
+  1. Provider service, [cardano-services](./packages/cardano-services/README.md): a REST API back-end that can query the Cardano Blockchain and submit transactions.
+  1. Client libraries meant to help with concepts like: key management, cryptography high level helpers, input selection, and many more.
```

Comment:
All (besides a couple exceptions like `web-extension`) libraries are suitable for server-side as well as client-side
```suggestion
  1. Libraries meant to help with concepts like: key management, cryptography high level helpers, input selection, and many more.
```

Reply:
[c2bbc7f](https://github.com/input-output-hk/cardano-js-sdk/pull/689/commits/c2bbc7faf956b36b8f1d59cff86343df83506eed)